openff/interchange/operations/_combine.py

```diff
@@ -44,6 +44,31 @@ def _check_nonbonded_compatibility(
             f"{interchange1['vdW'].switch_width} and {interchange2['vdW'].switch_width}.",
         )
 
+    if interchange1["vdW"].scale_14 != interchange2["vdW"].scale_14:
+        raise UnsupportedCombinationError(
+            "1-4 scaling factors in vdW handler(s) do not match.",
+        )
+
+    if interchange1["Electrostatics"].scale_14 != interchange2["Electrostatics"].scale_14:
+        if sorted([interchange1["Electrostatics"].scale_14, interchange2["Electrostatics"].scale_14]) == [
+            0.833333,
+            0.8333333333,
+        ]:
+            warnings.warn(
+                "Found electrostatics 1-4 scaling factors of 5/6 with slightly different rounding "
+                "(0.833333 and 0.8333333333). This likely stems from OpenFF using more digits in rounding 1/1.2. "
+                "The value of 0.8333333333 will be used, which may or may not introduce small errors. ",
+                InterchangeCombinationWarning,
+            )
```

Comment:
Here's where I'd like to link to upstream https://github.com/openforcefield/openff-forcefields/issues/120